packages/@aws-cdk/aws-servicecatalog/test/integ.product.js.snapshot/integ-servicecatalog-product.template.json

```diff
@@ -44,6 +44,15 @@
        }
       }
      },
+     {
+      "DisableTemplateValidation": false,
+      "Info": {
+       "LoadTemplateFromURL": {
+        "Fn::Sub": "https://s3.${AWS::Region}.${AWS::URLSuffix}/cdk-hnb659fds-assets-${AWS::AccountId}-${AWS::Region}/dd2d087eeb6ede1d2a9166639ccbde7bd1b10eef9ba2b4cb3d9855faa4fe8c1f.json"
+       }
+      },
+      "Name": "v1"
+     },
```

Comment:
This test doesn't make sense to me. How come we have the same artifact four times and twice with the same version attached to it?